contribs/gmf/src/directives/search.js

```diff
@@ -205,6 +208,17 @@ gmf.SearchController = function($scope, $compile, $timeout, gettextCatalog,
   this.timeout_ = $timeout;
 
   /**
+   * @type {ngeo.Location}
+   * @private
+   */
+  this.ngeoLocation_ = $injector.get('ngeoLocation');
+
+  /**
+   * @private
+   */
+  this.fullTextSearch_ = $injector.get('gmfFulltextSearchService');
```

Comment:
Now because this is a "strong" dependency (can't work without), please don't use the injection but add in the constructor parameter (just like `gmfTreeManager`)